holidays/countries/sao_tome_and_principe.py

```diff
@@ -0,0 +1,147 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.constants import PUBLIC
+from holidays.groups import ChristianHolidays, InternationalHolidays, StaticHolidays
+from holidays.observed_holiday_base import (
+    ObservedHolidayBase,
+    SUN_TO_NEXT_MON,
+    SAT_TO_PREV_FRI,
+)
+
+
+class SaoTomeAndPrincipe(
+    ObservedHolidayBase, ChristianHolidays, InternationalHolidays, StaticHolidays
+):
+    """São Tomé and Príncipe holidays.
```

Comment:
```suggestion
    """Sao Tome and Principe holidays.
```
I recommend using [ISO name spelling](https://www.iso.org/obp/ui/#iso:code:3166:ST) everywhere.